### PR TITLE
fix: Rename tag in profile page.

### PR DESCRIPTION
**Pull request by Anupam-dagar**
Reference: #6097

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Anupam-dagar** [comment]
@CleverFool77 I wasn't able to find js reference for most common tags and hence haven't made any js changes. Please review.

**Comment by Anupam-dagar** [comment]
yes, I am able to add the tags. Although a refresh needs to be done for the tags to be shown.

**Comment by CleverFool77** [comment]
And that's what we need to look into. Its the guess that because of same id for `most -commonly tags` and `user tags` is creating problem. Though I haven't checked this problem yet. 
What should be actually there ? -
The tags should be added as soon as we added like we used to have it earlier. There shouldn't be any kind of refresh needed.

**Comment by CleverFool77** [comment]
You need to look into this matter. This might be because of same `id-ns=ame` or there could be another reason.

**Comment by CleverFool77** [comment]
Then the purpose of this PR might get changed. and fix could be completely different.

**Comment by Anupam-dagar** [comment]
Sure, I'll look into the issue of why it is not getting inserted without the refresh and submit a fix.

**Comment by Anupam-dagar** [comment]
Hi @CleverFool77 , I have updated my PR, this makes it possible to show the tags in realtime as soon as it is added.

**Comment by Anupam-dagar** [comment]
@CleverFool77 Here is the gif showing the addition of tags.
Peek 2019-08-30 17-37

**Comment by SidharthBansal** [comment]
Great collab this time guys. I really loved the way you all are working in teams for solving bugs. PR backlog in April was too high. Now almost all prs are reviewed. This is great. Work product is amazing!!!!
Just checking this up and merging it as it is already approved!!! :-)

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -72,7 +72,7 @@
   </div>
 
   <hr>
-  
+
```
Comment (SidharthBansal) [inline app/views/users/profile.html.erb]: Quick tip: Remove unwanted lines of code from the PR before code reviews. This helps in error free quick pr reviews.
Thanks

**Pull request merged**